tailwind.config.js

```diff
@@ -0,0 +1,111 @@
+/** @type {import('tailwindcss').Config} */
+module.exports = {
+  content: ['./pages/**/*.{js,ts,jsx,tsx}', './components/**/*.{js,ts,jsx,tsx}'],
+  theme: {
+    extend: {
+      backgroundImage: {
+        'hero-underline-1': 'url(/icons/small-line.svg)',
+        'hero-underline-2': 'url(/icons/large-line.svg)',
+      },
+      backgroundPosition: {
+        under: 'bottom 0px right 15px',
+      },
+      colors: {
+        primary: {
+          100: '#E5E5E5',
+          200: '#CBCBCB',
+          300: '#B1B1B1',
+          400: '#979797',
+          500: '#7D7D7D',
+          600: '#636363',
+          700: '#494949',
+          800: '#2E2E2E',
+          900: '#141414',
+        },
+        secondary: { 400: '#9A9A9A' },
+      },
+      fontSize: {
+        3.75: '0.9375rem',
+        4.75: '1.1875rem',
+        7.5: '1.875rem',
+        9.5: '2.375rem',
+        11.75: '2.9375rem',
+        14.75: '3.6875rem',
+        18.25: '4.5625rem',
+        23: '5.75rem',
+      },
+      maxWidth: {
+        '1/2': '50%',
+        '1.05/2': '55%',
```

Comment:
these fractions should actually equal the value of the percentage. meaning that (1.02/2) * 100 should equal `55`. 

But here the value here is `51%`. 


A quick math formula to convert a percentage to a fraction is to divide the percentage by `100` and add a denominator of `100`.

For example, if you want to convert 25% to a fraction, you would divide `25` by `100` and get `0.25`. Then you can add a denominator of `100`, to get the fraction `25/100`.

You will have to simplify both the numerator and denominator to the lowest possible value without a floating number. In this example, we will simplify it to 1/4 by dividing both the numerator and denominator by 25.

In our case, we will take `55/100` and simplify it to `11/20` by diving both the numerator and denominator by 5.

You can use this link to help with the conversion https://www.calculatorsoup.com/calculators/math/percent-to-fraction-calculator.php